### DISCUSSION TITLE
Proper way to type field errors

### DISCUSSION THREAD

**Discussion by zizhengtai** [react-hook-form/react-hook-form#2256] [Q&A]
Hi all, I'm writing a cutom resolver, and I noticed the `FieldErrors` type isn't exposed in the public API. I have a helper function that looks like this:
```ts
const getFieldErrors = <Values>(err: MyError): object => {
  ..
};
```
...and ideally I'd want it to return something like `FieldErrors<Values>`.

Is `FieldErrors` the right type to use here?

**Comment by bluebill1049** [comment]
can you leverage: https://react-hook-form.com/ts#FieldError?
